job-server/src/spark.jobserver/JobServer.scala

```diff
@@ -26,6 +26,13 @@ import org.slf4j.LoggerFactory
 object JobServer {
   val logger = LoggerFactory.getLogger(getClass)
 
+  private def setSystemPropertiesFromConfig(config: Config) {
```

Comment:
I move the setter / getter code for this into a new utility object. JobServer now calls: 
Config2Properties.setSystemPropertiesFromConfig(config)
and the LdapGroupRealm reads it with:
private val searchBase: String = Config2Properties.getSystemProperty(Config2Properties.LDAP_SEARCH_BASE_PARAM)

That minimizes changes to the JobServer and quarantines the ugly implementation detail to Config2Properties.